app/src/main/java/com/greenfox/opal/gitinder/model/response/SwipingResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.greenfox.opal.gitinder.model.response;
+
+/**
+ * Created by Nagy Dóra on 2017.06.22..
+ */
+
+public class SwipingResponse extends BaseResponse {
+
+	private boolean match_status;
```

Comment:
@dorinagy the json converter is able to convert from `camel_case` to `camelCase` and vica versa if needed, so we should use the latter style